src/common/__tests__/timezones.test.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+describe("Timezones", () => {
+  it("should always be UTC", () => {
+    expect(new Date().getTimezoneOffset()).toBe(0);
```

Comment:
Why you're testing standard JS lib and moreover the test should be running only at server with UTC==0?

Reply:
Only requirement for this test suite would be to act as documentation. It should carefully explain why we are faking timezone (and time, check other comment about setting determistic current time in test setup). 

Currently name of test doesn't give any explanation why we are doing this.

Reply:
This test isn't deterministic. It fails e.g. on my computer.